src/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Wbpack Exercise</title>
+    
+</head>
+<body>
+<main>
+    <div class="listElement">
+        <h1>Today's To Do</h1>
+    </div>
+    <div class="listElement">
+        <form action="#">
+            <input type="text" placeholder="Add to your list...">
+        </form>
+    </div>
+    
+    <ul id="todolist" class="listElement">
+    </ul>
+    <button type="button" class="listElement">Clear all completed</button>
+    <script src="main.js"></script>
+</main>
+</body>
+</html>
```

Comment:
- [Optional] Consider using an indentation of two spaces in all your documents. This is a convention and also [best practice.](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)